renovate.json5

```diff
@@ -22,6 +22,17 @@
     "regexManagers:githubActionsVersions",
   ],
 
+  // https://docs.renovatebot.com/configuration-options/#ignorepresets
+  ignorePresets: [
+    // https://docs.renovatebot.com/presets-default/#ignoremodulesandtests
+    // By default, some directories, include "tests", are ignored. Since we do want Renovate to pick up some dependencies
+    // in "tests" directory, we explicitly opt out from this preset, instead ignoring specific subdirectories.
+    ":ignoreModulesAndTests",
+  ],
+
+  // https://docs.renovatebot.com/configuration-options/#ignorepaths
+  ignorePaths: ["tests/fixtures/**"],
```

Comment:
As specified in the PR description, we could in the future unignore dependencies in fixtures, to let Renovate update dependencies on a regular basis.